source/_components/light.blinkt.markdown

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: "Blinkt!"
```

Comment:
Oh if that's the name standardized the other way around and try to use Blinkt! everywhere possible. 😊